tests/artist-create.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db/index');
+const app = require('../src/app');
+const { createArtist } = require('../src/controllers/artist');
+const sinon = require('sinon');
+
+describe('Create artist', () => {
+  describe('/artists', () => {
+    describe('POST', () => {
+      it('expects a 201 status code', async () => {
+        const req = {};
+        const res = { status: sinon.spy()};
+
+        createArtist(req, res);
+
+        // expect(res.sendStatus).to.equal(201);
+        // expect(res.status.calledOnce).to.be.true;
+        // expect(res.status.calledWith(201)).to.be.true;
```

Comment:
This should be fixed. Rather than commenting out assertions and allowing the test to (falsely) pass, its better to put an 'x' before the 'it' keyword above so that when the tests are run, they are shown as skipped and better visibility is given.